### ISSUE TITLE
Log4j2 not working with probe

### ISSUE THREAD

**Issue by GoogleCodeExporter** [psi-probe/psi-probe#408]
```
What steps will reproduce the problem?
1.Create a an web application that implements slf4j and log4j2 as its logging 
framework
2.Deploy probe to tomcat.
3.And try to access probe and access the logs created by Log4j2 framework, it 
is not there

What is the expected result?
The logs should show along with jdk logs(catalina.out) in /probe/logs/index.htm 
page

What happens instead?
All the logs created by Log4j2 framework are not showing up in the logs 
page(/probe/logs/index.htm)

What version of Probe are you using?
2.3.3

What environment (browser version, Tomcat version, JVM version, server OS)?
Browsers - All, Tomcat Version - 6.0.37, Java 1.6.0-35, Linux Ubuntu 13.04


Please provide any additional information below.
(Attach logs or stack traces as files instead of pasting the contents
here.)

```

Original issue reported on code.google.com by `Sabin.Ga...@gmail.com` on 9 May 2014 at 3:16

**Comment by oers** [comment]
Is there documentation on "How To write a logger wrapper"? Log4j2 support would be great.

**Comment by j-renkel** [comment]
log4j2 is now at version (at least) 2.6, and is extremely stable. We are converting all of our webapps to use it (many reasons), and would greatly appreciate support for it in probe

**Comment by hazendaz** [comment]
@j-renkel This code was just added.  Can you build off master and give it a try to see if it solves log4j2 needs?  I'm going to close this issue now.  If you run into any problems, please do open a new ticket.

**Comment by j-renkel** [comment]
My manager finally allocated some time for me to look into this.

Turns out log4j2 has several ways of initializing and storing its logging configuration, and the existing code only supported the simplest of them.

I've revised the code to, I believe, support all of them. I've tested as many as I could in our tomcat environment, but there may be others I haven't tested.

I will gladly contribute this code to the community.

Can I initiate a pull request against this closed issue? Or do I need to open a new issue for the pull request?

Thanks in advance,

Jim Renkel
AMI Entertainment Network, Inc, LLC
